release/release.md

```diff
@@ -0,0 +1,112 @@
+# How to do a Kata Containers Release
+
+Hi if you are reading this document you may also want to create a kata Containers Release.
+
+The Kata Containers Release Process is defined in the follwoing documents.
+
+https://github.com/kata-containers/documentation/blob/master/Releases.md
+
+To simply the process of read each release we have created a checklist for it.
```

Comment:
s/of read each/???/  I'm not quite sure what you meant here, but I suspect it's a typo.

Reply:
Nit: s/follwoing/following